src/NetSuiteAccess/Models/SalesOrder.cs

```
@@ -169,7 +169,7 @@ public static NetSuiteSalesOrder ToSVSalesOrder( this NetSuiteSoapWS.SalesOrder
 					{
 						Quantity = (int)Math.Floor( itemInfo.quantity ),
 						Sku = itemInfo.item != null ? itemInfo.item.name : string.Empty,
-						UnitPrice = itemInfo.rate != null ? decimal.Parse( itemInfo.rate, System.Globalization.NumberStyles.Any, CultureInfo.InvariantCulture ) : 0,
+						UnitPrice = (decimal) ( itemInfo.amountSpecified ? Math.Round( itemInfo.amount / itemInfo.quantity, 2 ) : 0 ),

```

Should we validate that quantity is > 0, so that if it isn't we can throw a clearer error ("quantity is 0"), then one  about UnitPrice having divide by 0? A 0 quantity would be abnormal, but things happen

---

@maxim-skuvault You're right, just tried to specify line item in NetSuite with zero quantity and NetSuite successfully saved the changes.